Rubberduck.Parsing/VBA/ReferenceResolveRunnerBase.cs

```diff
@@ -202,7 +210,8 @@ private Declaration SupertypeForDocument(QualifiedModuleName module, RubberduckP
 
         protected void ResolveReferences(DeclarationFinder finder, QualifiedModuleName module, IParseTree tree, CancellationToken token)
         {
-            Debug.Assert(_state.GetModuleState(module) == ParserState.ResolvingReferences || token.IsCancellationRequested);
+            // This assert is no longer true now that we throw resolver error when resolving references
+            // Debug.Assert(_state.GetModuleState(module) == ParserState.ResolvingReferences || token.IsCancellationRequested);
```

Comment:
Thanks for the explanation. However, I would prefer if the commented out code simply vanished from the code.

Reply:
OK. I will remove; I just wanted to be sure I was not violating some other assumptions I might not have had considered.